chart/templates/support-bundle.yaml

```diff
@@ -54,81 +45,30 @@ stringData:
                   args: ["-exc", "npm run ready-to-run:built"]
                   env:
                     - name: WORKSPACE_POSTGRES_PASSWORD
-                      value: {{ .Values.externalPostgresql.password | quote }}
-                      # valueFrom:
-                      #   secretKeyRef:
-                      #     name: {{ include "carto.postgresql.secretName" . }}
-                      #     key: {{ include "carto.postgresql.secret.key" . }}
+                      valueFrom:
+                        secretKeyRef:
+                          name: {{ include "carto.postgresql.secretName" . }}
```

Comment:
I'm worried with that. What happens if the apply failed and the client run the support bundle? the secret/configmap is in another version (or could be not present). 

Also, if we put as a preflight (something that should be done before or after), the config file is not going to be created, right? If you couldn't think a better way, please ask
[in the shared slack channel](https://cartoteam.slack.com/archives/C05V5V84W7M)

Reply:
Okay, in that case we have to obtain these values (secrets and config) from the customer customizations fields. I'm doing the refactor